sdlpp.hpp

```diff
@@ -400,6 +400,19 @@ namespace sdl
     METHOD(blit, BlitSurface)
     METHOD(softStretch, SoftStretch)
     METHOD(blitScaled, BlitScaled)
+
+    class Lock {
+      public:
+        explicit Lock(Surface& s) : m_s(s) {
+          if (SDL_MUSTLOCK(s.get())) s.lock();
+        }
+        ~Lock() {
+          if (SDL_MUSTLOCK(m_s.get())) m_s.unlock();
+        };
+
+      private:
+        Surface& m_s;
```

Comment:
I do not like `m_` prefixes. UwU, can you remove it, plz? 
```suggestion
        Surface& s;
```